index.bs

```diff
@@ -475,7 +475,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     1. If |extension| is not supported by this client platform, then [=continue=].
 
     1. Otherwise, let |result| be the result of running |extension|'s [=client processing=] algorithm on |argument|. If the
-        algorithm returned an error, [=continue=].
+        algorithm returned an error, return [=a promise rejected with=] a {{DOMException}} whose name is
+        "{{TypeError}}", and terminate this algorithm.
```

Comment:
I can understand the argument that a programmer error (such as a badly formed client argument to an extension) should be flagged. However, this has a couple of problems:
1. If the specification of an extension evolves in future (e.g. extension adds a new version and the version number is given in the client argument) then this will either leak data or break an operation that could have been completed correctly by ignoring the extension.
1. Client processing can also fail for a number of other reasons other than malformed arguments (e.g. authenticator selection not finding authenticators of the requested type). In those cases this error becomes misleading and also leaks privacy-sensitive info (which can be used to fingerprint the system).